mass_mailing_list_dynamic/models/mail_mass_mailing_list.py

```diff
@@ -43,22 +42,29 @@ def action_sync(self):
         # Skip non-dynamic lists
         dynamic = self.filtered("dynamic")
```

Comment:
I wonder why this doesn't fail without this patch :thinking::

```suggestion
         dynamic = self.filtered("dynamic").with_context(syncing=True)
```